index.d.ts

```diff
@@ -179,6 +179,21 @@ export interface Options {
 	*/
 	readonly removeQueryParameters?: ReadonlyArray<RegExp | string> | boolean;
 
+	/**
+	Keeps query parameters that matches any of the provided strings or regexes.
+
```

Comment:
It needs a note that says that this option overrides `removeQueryParameters`.

Reply:
Added (c21c0cb22e7ae5be486e79123f12344af7482bde)

Also updated README.md based on the first and third your change requests (cf23009201f01f8b01db44737d1ebbf58bd672e0)